test/test_vmap.py

```diff
@@ -2548,6 +2548,34 @@ def test_conv2d(self):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(conv_fn, arg_values, kwarg_values):
                 self.assertEqual(loop_out, batched_out)
 
+    def test_grid_sample2d(self):
+        N, C, DIn, HIn, WIn, DOut, HOut, WOut = range(1, 9)
```

Comment:
More tests are always good, but are these testing anything that isn't covered by the OpInfo tests?

Reply:
This is based on the one above for conv2d right above it (we could probably condense these with OpInfo?) . As I understood (which might be wrong), the OpInfo tests that already exist just test that the function didn't call the fallback and this tests correctness of the output (comparing a looped version to the vmap version)

Reply:
There are a bunch of OpInfo tests :P

We have some OpInfo tests that just test whether it has a batching rule (`test_op_has_batch_rule`), and some OpInfo tests that test actual correctness (`test_vmap_exhaustive`).

I think those conv2d tests were added when we didn't have a conv2d OpInfo.